config/migrate_test.go

```diff
@@ -64,6 +64,46 @@ func TestMigrateDatabaseToFile(t *testing.T) {
 	assert.Equal(t, ds.Get(), fs.Get())
 }
 
+func TestMigrateDatabaseToFileWithEmptyFiles(t *testing.T) {
+	if testing.Short() {
+		t.SkipNow()
+	}
+	helper := testlib.NewMainHelper()
+	sqlSettings := helper.GetSQLSettings()
+	fileDSN := "config.json"
+	files := []string{"", "", ""}
+	ds, err := NewDatabaseStore(getDsn(*sqlSettings.DriverName, *sqlSettings.DataSource))
+	defer ds.Close()
```

Comment:
This will panic if NewDatabaseStore returned an error.  I know the test harness will capture the panic, but it would be cleaner to no panic here IMO.

Reply:
As Doug rightly pointed out: error checks first, and then defers.